src/web/mjs/engine/DownloadJob.mjs

```diff
@@ -124,7 +124,7 @@ export default class DownloadJob extends EventTarget {
         for(let page of pages) {
             await this._wait(this.throttle);
             const response = await fetch(page, this.requestOptions);
-            if(response.status !== 200) {
+            if(!(response.status === 200 || response.status === 404 && Engine.Settings.ignore404.value)) {
```

Comment:
The feature should be "ignore errors" and not just 404.
404 seems to be for a specific connector case
Something like that 
`          if(response.status !== 200 && !Engine.Settings.ignoreError.value)) {`

Reply:
Updated it!

Reply:
don't use not OR, use AND, it's easier to read

Reply:
Fixed